remindme-common/src/main/java/it/andreafailli/remindme/TestCategories.java

```diff
@@ -0,0 +1,5 @@
+package it.andreafailli.remindme;
+
+public class TestCategories {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this empty class, write its code or make it an "interface". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2094)